### PR TITLE
Improve synchronization between parent/child/grandchild

### PR DESCRIPTION
**Pull request by yyyeerbo**
Uses raw bytes to synchronize between processes, serde/serde_json
is cumbersome to do it.

Enhancement: #53

Signed-off-by: Yang Bo <bo@hyper.sh>

### CODE REVIEW

**Comment by yyyeerbo** [comment]
@lifupan @john-intel

File: rustjail/src/container.rs [hunk 1]
```diff
@@ -786,6 +786,58 @@ fn write_json(fd: RawFd, msg: &SyncPC) -> Result<()>
 	Ok(())
 }
 
+pub const PIDSIZE: usize = 4;
```
Comment (lifupan) [inline rustjail/src/container.rs]: Is is better to use mem::size_of::<pid_t>() to calculate the pid size instead of hardcore it?

Comment (yyyeerbo) [reply rustjail/src/container.rs]: array  can be in variable size? Or use vec?

Comment (yyyeerbo) [reply rustjail/src/container.rs]: I really don't like this kind of code
for e in v.iter() {
   pid = (pid << 8) + *e as pid_t;
}
to replace 
pid_t::from_be_bytes(array)

Comment (lifupan) [reply rustjail/src/container.rs]: > array can be in variable size? Or use vec?

mem::size_of<>() is a const fn now, and it should be executed at compile time, thus you can replace the value '4' with it directly.

**Pull request merged**